k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -2069,6 +2069,10 @@ spec:
                 tlsPrivateKeyFile:
                   description: 'TODO: Remove unused TLSPrivateKeyFile'
                   type: string
+                topologyManagerPolicy:
+                  description: TopologyManagerPolicy determines the policy for the
+                    topology manager
```

Comment:
```suggestion
                  description: TopologyManagerPolicy determines the allocation policy for the
                    Topology Manager
```

Reply:
Hehe. Thanks :)

Reply:
I think this should have been changed in the 2 `componentconfig.go` files. At the moment seems reverted.

Reply:
yeah. I made a mistake there. Should be better now.

Reply:
Thanks :)